frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/deprecated-operator-warnings.cy.ts

```diff
@@ -0,0 +1,140 @@
+import { checkErrors, create } from '../../../integration-tests-cypress/support';
+import { detailsPage } from '../../../integration-tests-cypress/views/details-page';
+import { nav } from '../../../integration-tests-cypress/views/nav';
+import { testDeprecatedCatalogSource, testDeprecatedSubscription } from '../mocks';
+
+const WAIT_TIME = 30000;
```

Comment:
This value looks good in the local environment. It could change depending on how the installation behaves in CI.